src/main/java/com/rultor/profiles/Profiles.java

```diff
@@ -73,18 +79,77 @@ public Profile fetch(final Talk talk) throws IOException {
             if (xml.nodes("/talk/wire").isEmpty()) {
                 profile = Profile.EMPTY;
             } else {
-                profile = Profiles.fetch(xml);
+                profile = this.fetch(xml);
             }
         }
         return profile;
     }
 
     /**
+     * Merge profile from master and fork. Merged profile must be profile
+     * from fork, but all lists for commanders and architects must be taken
+     * from master branch.
+     * @param master Profile from master branch
+     * @param fork Profile from fork branch
+     * @return Merged profile
+     * @todo #1039:30min Modify method implementation: merged profile is a
+     *  `.rultor.yml` form result of real `git merge` of master branch and
+     *  fork branch. We need real `git merge` result here. See some details
+     *  here: https://github.com/yegor256/rultor/pull/1064/files#r56796959
+     */
+    public Profile merged(final Profile master, final Profile fork) {
```

Comment:
@xupyprmv This is the only slight issue in this PR that I can find :)
Why do we need the fork profile here ? :)

In my opinion we don't because as you explain just fine in the Puzzle, the actual merge is going to happen on the EC2 instance using actual GIT :) That thing only needs to know the name of the fork (the coordinates so to say ;)).
I suggest simply making the second input the coordinate string here. Then we can pass that to GIT => lots easier :)
I'll explain one more detail on this below ;) Here really just make the second input `final String fork`.